businesslogic/analytics-publish-documents.ts

```diff
@@ -56,7 +58,24 @@ export const getAnalyticsProcessorForDocuments = <I>(
             RA.map(success => success.document),
             mainPublisher.publishes,
             T.map(RA.concat(enrichResults)),
-            T.map(RA.filter(isFailure))
+            T.map(RA.filter(isFailure)),
+            T.map(
+              RA.map(failure =>
+                pipe(
+                  enablePublishTracking,
```

Comment:
We could move the 'enablePublishTracking' in the creation of the tracker-outbound-publisher and evaluate it in the 'trackEvent' method